src/pages/Login/Login.js

```diff
@@ -1,7 +1,148 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AlertCircle, ArrowLeft, Eye } from 'react-feather';
+import './Login.scss';
 
 function Login() {
-  return <div>Login.js</div>;
+  const [inputs, setInputs] = useState({
+    userId: '',
+    password: '',
+  });
+
+  const { userId, password } = inputs;
+
+  const navigate = useNavigate();
+
+  const [passwordVisible, setPasswordVisible] = useState(false);
+
+  const [warning, setWarning] = useState({
+    idIncorrect: false,
+    pwIncorrect: false,
+  });
+
+  const { idIncorrect, pwIncorrect } = warning;
+
+  const handleInput = event => {
+    const { name, value } = event.target;
+    event.preventDefault();
+
+    setInputs({
+      ...inputs,
+      [name]: value,
+    });
+  };
+
+  const handleClick = () => {
+    setPasswordVisible(!passwordVisible);
+  };
+
+  const handleSubmit = () => {
+    allInputValid ? console.log(inputs) : alert('정보값을 확인해주세요!');
+  };
+
+  const userIdValid = userId.match(/^\S+@\S+\.\S+$/);
+  const passwordValid = password.match(/^(?=.*[A-Za-z])(?=.*[0-9])(?=.{8,20})/);
+
+  const allInputValid = userIdValid && passwordValid;
+
+  return (
+    <div className="login">
+      <div className="left">
+        <div className="column-content">
+          <div className="column-top">
+            <ArrowLeft className="arrow-left" />
+            <div className="wekea-logo-white" />
+          </div>
+          <div className="hero-wrapper">
+            <h1 className="title">로그인</h1>
+            <p>
+              외워야 할 비밀번호가 많아 불편하셨죠? 이제 일회용 코드를 이용하여
+              간편하게 로그인하세요.
+            </p>
+            <p>* 이메일 또는 휴대폰 번호 최초 인증 후 사용 가능</p>
+          </div>
+          <span>
+            <div className="kiosk-label">
+              WEKEA.kr - 쿠키 정책 , 개인정보처리방침
+            </div>
+            <div className="kiosk-label">
+              © Inter WEKEA Systems B.V. 1999-2023
+            </div>
+          </span>
+        </div>
+      </div>
+      <div className="right">
+        <div className="right-content-container">
+          <label>
+            이메일 또는 확인된 핸드폰 번호
+            <input
+              onChange={handleInput}
+              onBlur={() => {
+                !userIdValid
+                  ? setWarning({ ...warning, idIncorrect: true })
+                  : setWarning({ ...warning, idIncorrect: false });
+              }}
```

Comment:
```suggestion
              onBlur={() => {
                handleWarningMsg(userIdValid, 'idIncorrect');
              }}
```

해당 부분도 인자를 전달하는 방식으로 구현할 수 있겠습니다!
한번 이 구조를 보시고 handleWarningMsg 함수를 어떻게 구현해야할 지 한번 고민해보시죠!

Reply:
인자를 넘겨주는 방식으로 변경하긴 했는데, 맞게 한건지 모르겠습니다!!